daemons/controld/controld_callbacks.c

```diff
@@ -205,14 +205,14 @@ peer_update_callback(enum crm_status_type type, crm_node_t * node, const void *d
                  * transient attributes intact until it rejoins.
                  */
                 if (compare_version(fsa_our_dc_version, "3.0.9") > 0) {
-                    erase_status_tag(node->uname, XML_TAG_TRANSIENT_NODEATTRS, cib_scope_local);
+                    update_attrd_clear_node(node->uname, is_remote);
                 }
 
             } else if(AM_I_DC) {
                 if (appeared) {
                     te_trigger_stonith_history_sync();
                 } else {
-                    erase_status_tag(node->uname, XML_TAG_TRANSIENT_NODEATTRS, cib_scope_local);
+                    update_attrd_clear_node(node->uname, is_remote);
```

Comment:
the current goal is to avoid that (T139). the plan is that the controller won't request attribute removal, the attribute manager will do it when a node leaves the cluster layer